src/renderer/state.ts

```diff
@@ -536,6 +542,7 @@ export class AppState {
       this.save(GlobalSetting.acceleratorsToBlock, this.acceleratorsToBlock),
     );
     autorun(() => this.save(GlobalSetting.packageAuthor, this.packageAuthor));
+    autorun(() => this.save(GlobalSetting.isShowingGistHistory, false));
```

Comment:
issue(blocking): I think this conflicts with the following snippet from line 121 above:

```ts
  public isShowingGistHistory = !!(
    this.retrieve(GlobalSetting.isShowingGistHistory) ?? true
  );
```

From my testing, the first load shows the Gist history feature but subsequent loads turn the setting off without explicitly doing so.